src/main/java/frc/robot/subsystems/CoralRoller.java

```diff
@@ -78,16 +88,21 @@ public class CoralRoller extends SubsystemBase implements ActiveSubsystem, Shuff
           METERS_PER_REVOLUTION);
   private final RelativeEncoder encoder = motor.getEncoder();
   private DigitalInput beamBreak = new DigitalInput(CORAL_ROLLER_BEAM_BREAK);
+  private LaserCan laserCAN = new LaserCan(CAN.CORAL_ARM_LASER_CAN);
 
   private final SimpleMotorFeedforward feedforward = new SimpleMotorFeedforward(KS, KV);
   private final PIDController pidController = new PIDController(1, 0, 0);
   private final Timer outtakeTimer = new Timer();
 
+  public static boolean detectsReef = false;
```

Comment:
`detectsReef` should be an instance variable and not `static`.
```suggestion
  public boolean detectsReef = false;
```

Reply:
Agreed